### PR TITLE
Fix problem where null library volume changed to zero (GLT-1137)

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/library_hot.js [hunk 1]
```diff
@@ -619,7 +619,7 @@ Library.hot = {
       if (obj.volume) {
         lib.volume = obj.volume;
       } else {
-        lib.volume = 0;
+        lib.volume = null;
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/library_hot.js]: If obj.volume is 0 here, it ends up getting set to null. 0 should be a savable value as well though

**Comment by apmasell** [comment]
Fixed.

**Comment by djcooke** [comment]
Now volume is always 1.0 on load of Edit Library (bulk) page, whether the volume is actually null, 0, or 1. If you change it and save, it saves correctly, but then always displays 1.0 on load still. Single Edit page works fine

**Comment by apmasell** [comment]
That's once again due to Spring poisoning the cache by changing the model object which is also the cached copy.

**Pull request merged**